props/saber_BC_buttons.h

```diff
@@ -2042,6 +2064,154 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
 
 #endif  // BC_DUAL_BLADES
 
+  // Manual blade array selection
+  // even with Real TIme Blade ID active
+  bool manual_blade_array_active = false;
+  size_t last_real_best_config = NELEM(blades);
```

Comment:
Each of these variables need a comment to explain what they do and why.

NELEM(blades) is probably not a valid value for these variables, I assum that is on purpose?

Reply:
changed to SIZE_MAX